src/components/UserItem.js

```diff
@@ -0,0 +1,23 @@
+function UserItem({ key, user }) {
```

Comment:
There's no need to pass the `key` as a prop to the `UserItem` component, the `key` prop will be used inside the `UserList` component instead when you map over each `UserItem`.

The `key` prop is going to be used by react internally and passing it to the component itself won't be of any use.

e.g:
```js
// inside the UsersList component
// ...
{
                        usersList.map((user) => {
                            return <UserItem key={user.id} user={user} />
                        })
                    }
```
---
```js
// inside UserItem
function UserItem({ user }){
    return (
        <tr>
            <td class="border-b border-gray-200 bg-white px-5 py-5 text-sm">
                <p class="whitespace-no-wrap">{user.name}</p>
            </td>
            <td class="border-b border-gray-200 bg-white px-5 py-5 text-sm">
                <p class="whitespace-no-wrap">{user.email}</p>
            </td>
            <td class="border-b border-gray-200 bg-white px-5 py-5 text-sm">
                <p class="whitespace-no-wrap">{user.phone}</p>
            </td>
            <td class="border-b border-gray-200 bg-white px-5 py-5 text-sm">
                <p class="whitespace-no-wrap">{user.company.name}</p>
            </td>
        </tr>

    )
}
```